css/css-cascade/important-prop.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>CSS Cascade: Important</title>
+  <link rel="author" title="David Burns" href="http://www.theautomatedtester.co.uk">
+  <link rel="help" href="https://drafts.csswg.org/css-cascade/#importance">
+  <link rel="match" href="important-prop-ref.html">
+  <meta name="flags" content="">
+  <style>
+  @keyframes override{
+   from, to{background-color: #f00;}
+  }
+
+  .square {
+    color:#00f;
+    animation: override 1s infinite;
+    width: 100px;
+    height: 100px;
+  }
+  .green {
```

Comment:
you don't need the two different classes do you?

Reply:
I do, unless you can suggest an alternative. What I want to show is that it should be green because of the `!important`. Currently on Chrome the box shows red and in other browsers it shows green.